data/faq.md

```diff
@@ -31,40 +53,40 @@ A. To get an idea of what to expect, please have a look at the
 [EP2021 programme](https://ep2021.europython.eu/events/sessions/) and the
 [EP2019 programme](https://ep2019.europython.eu/events/sessions/).
 
-To select talks, we run a Call for Proposals (CFP). Talks are then curated based
-on the input from the community voting and expert panel reviews. Make sure to subscribe to our
-[EuroPython Blog](https://blog.europython.eu/#/portal/signup) for updates.
+The conference will feature around 20 tutorials/workshops, 120 talk sessions and other interactive sessions such as Posters, Helpdesks, Panel Discussions and Maker Space.
```

Comment:
"and a Maker Space" (there's only one maker space, not several?)

Reply:
touché!